data/yam/agama/auto/lib/questions.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
+    questions():: {
```

Comment:
we would need a way to call this more specific, as we will have multiple questions answered in this file.